mirage-nat/unikernel.ml

```diff
@@ -69,25 +85,27 @@ module Main (R : Mirage_random.S) (M : Mirage_clock.MCLOCK) (P : Mirage_clock.PC
             | Ok packet -> output_tunnel packet
             | Error `Untranslated -> add_rule table packet
             | Error `TTL_exceeded ->
+              (* TODO should report ICMP error message to src *)
               Log.warn (fun f -> f "TTL exceeded"); Lwt.return_unit
```

Comment:
but it is not necessarily ICMP... the on the private interface received packet has a TTL of 1, i.e. it should not be routed further. this may be any IP packet...